marklogic-data-hub-central/ui/src/components/selected-facets/selected-facets.tsx

```diff
@@ -116,14 +117,14 @@ const SelectedFacets: React.FC<Props> = (props) => {
               <HCButton
                 size="sm"
                 variant="outline-light"
-                className={styles.dateFacet}
+                className={styles.facetButton}
                 key={index}
                 onClick={() => clearRangeFacet(item.constraint)}
                 data-cy={`clear-${item.rangeValues.lowerBound}`}
                 data-testid={`clear-${item.displayName}`}
               >
                 {facetName + ": " + item.rangeValues.lowerBound + " ~ " + item.rangeValues.upperBound}
-                <Icon type="close" />
+                <XLg className={styles.close}/>
```

Comment:
@federicoRufrancosMarklogic, similar comments about the padding for the facet containers in Explore. Make these consistent with what we see in Monitoring. Thanks!
<img width="460" alt="Screen Shot 2021-09-23 at 6 18 54 PM" src="https://user-images.githubusercontent.com/477757/134603968-cb1bd8bd-e553-499b-b6c0-6dacb64d302a.png">

Reply:
Same here, as I add some styling on the theme, these styling was also updated